cowprotocol/cowswap
# feat: handle toasts

# Summary

Continues on https://github.com/cowprotocol/cowswap/pull/4022

This PR handles TOAST events, and shows a snackbar. So now, CoW Swap will not show the toast, and instead the configurator will show the toast below, as it can be seen in the screenshot.

<img width="1230" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/7d0c5294-8950-4aa8-913e-91cff092efb0">


# To Test

1. Setup "Dapp mode" for the toasts

![image](https://github.com/cowprotocol/cowswap/assets/2352112/1ba971ab-bcf9-440c-8cd7-1ec3f25a0e86)


2. Check that the toast is shown when you do a trade


3. Setup "Self contained" mode for the toasts



3. Check now toasts show in the Dapp only (no snackbars)
